integreat_cms/cms/views/users/region_user_actions.py

```diff
@@ -36,9 +37,13 @@ def delete_region_user(request, region_slug, user_id):
 
     region = request.region
     user = get_object_or_404(region.region_users, id=user_id)
+
     if user.regions.count() == 1:
-        logger.info("%r deleted %r", request.user, user)
+        # Mark feedback read by the user as unread to prevent IntegrityError
+        Feedback.objects.filter(read_by=user).update(read_by=None)
```

Comment:
```suggestion
        user.feedback.update(read_by=None)
```
I think this would work as well, but doesn't make a huge difference.